src/orchestrator/src/lib.rs

```diff
@@ -196,7 +196,7 @@ pub struct ServiceConfig<'a> {
     /// A function that generates the arguments for each process of the service
     /// given the assigned listen addresses for each named port.
     #[derivative(Debug = "ignore")]
-    pub args: &'a (dyn Fn(&BTreeMap<String, String>) -> Vec<String> + Send + Sync),
+    pub args: Box<dyn Fn(&BTreeMap<String, String>) -> Vec<String> + Send + Sync>,
```

Comment:
This is necessary because of the process orchestrator, where we send this closure to the `OrchestratorWorker`. For the k8s orchestrator, we can already generate the args before we construct the `WorkerCommand`.

Reply:
Yeah, seems fine! None of this is perf sensitive at all.